brainiak/utils/simulator.py

```diff
@@ -0,0 +1,1058 @@
+#  Copyright 2016 Intel Corporation
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""Brain Simulator
+
+Simulate neural data for a single subject. This involves specifying what
+feature_type of noise and signal will be present in the data, as well as basic
+properties like the number of voxels and TRs to be simulated
```

Comment:
Could you give a reference for the simulator? If not, please explain in a few paragraphs what the simulator does.